tests/test_RetrievalEvaluator.py

```diff
@@ -13,7 +13,7 @@ def setup_method(self):
 
         setup_method is invoked for every test method of a class.
         """
-        self.evaluator = RetrievalEvaluator()
+        self.evaluator = RetrievalEvaluator(task_name="test")
```

Comment:
Like earlier comment, I think `task_name` should not be mandatory if there are cases where it is not needed like here.

Also, maybe we should have a test to ensure that prompting works? E.g. test it produces different results on a toy model with the latest sentence transformers or sth?

Reply:
Completely agree. I have added a test to ensure that the task_name is corrected and passed to models where it is relevant and does not influence the sentence transformer, which does not take the prompt.